gov_uk_dashboards/components/plotly/context_box.py

```diff
@@ -0,0 +1,29 @@
+"""Context_box"""
```

Comment:
Should this docstring and the filename be renamed to context_banner?

Reply:
Renamed to context_banner